wicked_pdf.gemspec

```diff
@@ -35,4 +35,6 @@ DESC
   spec.add_development_dependency 'sqlite3', '~> 1.3.6'
   spec.add_development_dependency 'mocha', '= 1.3'
   spec.add_development_dependency 'test-unit'
+  spec.add_development_dependency 'bootsnap'
+  spec.add_development_dependency 'rdoc'
```

Comment:
These are only dependencies on newer versions of Rails, and are included in https://github.com/mileszs/wicked_pdf/blob/master/gemfiles/5.2.gemfile and https://github.com/mileszs/wicked_pdf/blob/master/gemfiles/rails_edge.gemfile

Reply:
ahh